package.json

```diff
@@ -17,7 +17,7 @@
     }
   ],
   "license": "WTFPL",
-  "main": "src/index.js",
+  "main": "dist/angular-hal.js",
```

Comment:
The main property has to point to the index.js and not a generated dist file. If this doesn't work with your webpack config, I recommend changing the config.